ts-declaration/types.d.ts

```diff
@@ -743,7 +761,7 @@ export class PhoneCall {
         callInfo?: CallInfo;
         reason?: Constants.HANGUP_REASON;
         closeCallOnError?: boolean;
-        agentStatus?: Constants.HANGUP_STATUS;
```

Comment:
This was added intentionally, are there options missing from the HANGUP_STATUS enum?